### PR TITLE
Add YAML example for interpreters

### PR DESCRIPTION
**Pull request by alexvy86**
Adds an example of how interpreters are configured if using a YAML config file

### CODE REVIEW

**Review by alexvy86 — COMMENTED** [review]
Adds an example of how interpreters are configured if using a YAML config file

**Review by halostatue — APPROVED** [review]
It feels like we need to be a little clearer around line 116. Something like:

```markdown
Script interpreters can be added or overridden in the `interpreters`
section of the configuration file with *extension* keys.
```

After all, the TOML version *could* be done as:

```toml
[interpreters]
py = { command = 'C:\Python39\python3.exe' }
ps1 = { command = 'push', arguments = '-NoLogo' }
tcl = { command = "tclsh" }
```

**Comment by alexvy86** [comment]
@halostatue what do you think of that rewrite?

File: assets/chezmoi.io/docs/reference/target-types.md
Comment (halostatue) [inline assets/chezmoi.io/docs/reference/target-types.md:118]: I think that the `!!!note` is still useful, or something like it, because it makes it clear that the key for `.pl` is `pl`.

There are other changes that I would consider to show this, like the table:

```markdown
| Extension | Key   | Command      | Arguments |
| --------- | ----- | ------------ | --------- |
| `.pl`     | `pl`  | `perl`       | *none*    |
| `.py`     | `py`  | `python3`    | *none*    |
| `.ps1`    | `ps1` | `powershell` | `-NoLogo` |
| `.rb`     | `rb`  | `ruby`       | *none*    |
```

But that’s a bit more change than I would make without @twpayne agreeing. (Also, reading above, the bit about `%PATH%` should be `` in your `%PATH` or fully specified ``, but that’s *well* outside of the changes you’re contributing.

**Comment by bradenhilton** [comment]
We should probably use tabs like in https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#embedded-content at some point to give examples for each config format, but we certainly don't need to do it in this PR.

File: assets/chezmoi.io/docs/reference/target-types.md
Comment (alexvy86) [reply assets/chezmoi.io/docs/reference/target-types.md:118]: Re-added the note.

**Comment by alexvy86** [comment]
Ready for a last review. Someone from the team will merge it, right? Or if I should, just let me know; if so I'll wait for a last confirmation that it looks good.

**Pull request merged**
